docs/dietpi_tools.md

```diff
@@ -164,55 +168,55 @@ You can return to the **DietPi-Software** tool to make further changes at any ti
 
     ![DietPi-Config screenshot](assets/images/dietpi-config.jpg)
 
-    #### Display Settings
+    #### Display Options
 
     - Set your screen resolution, or go headless to save additional resources
     - Control your GPU memory splits
     - Enable/disable the RPi camera
 
-    **Audio Options**
+    #### Audio Options
 
     - Change sound cards with ease (e.g.: HiFiBerry / Odroid HiFi shield)
 
-    **Performance Options**
+    #### Performance Options
 
     - Overclock your system with a vast selection of overclocking profiles for your device
     - Change the CPU governor and tweak your ARM temperature limits
 
-    **Advanced Settings**
+    #### Advanced Settings
 
     - Configure swap file size
     - Update device firmware
     - Toggle serial console
     - Toggle Bluetooth
 
-    **Security Options**
+    #### Security Options
 
     - Change password and hostname
 
-    **Regional / Language Options**
+    #### Language/Regional Options
 
     - Set timezone, locale and keyboard options. Everything you'll need to make it feel like home
 
-    **Networking Options: Adapters**
+    #### Network Options: Adapters
 
     - Scan and connect to your WiFi router with ease
     - Change to a static IP address on your network
     - Configure your proxy settings
     - Test internet connection
     - Toggle IPv6 support
 
-    **Networking Options: NAS/Misc**
+    #### Network Options: NAS/Misc
```

Comment:
-> Network Options: Misc

Reply:
Da explizit auf die network drives eingegangen wird, hatte ich das "NAS" drin gelassen. Aber du hast recht, es ist schöner hier konsequent zu bleiben. Evtl. kann man auch deutlich machen, dass sich Samba und NFS mounts hinter "**Network Drives**" verbergen und das eigentlich nur zum **DietPi-Drive_Manager** weiterleitet. Dasselbe könnten wir in den Docs machen, anstatt das zu doppeln. Aber das kommt in einem separaten PR.

WTF, why I'm writing in German language? 😄